src/gurt/heap.c

```diff
@@ -20,6 +20,7 @@
  * Any reproduction of computer software, computer software documentation, or
  * portions thereof marked with this legend must also reproduce the markings.
  */
+
```

Comment:
any reason for these whitespace changes?

Reply:
Sorry, missed that